playbook/app/pb_kits/playbook/pb_table/docs/example.yml

```diff
@@ -26,6 +27,7 @@ examples:
     - table_striped: Striped Table
 
   react:
+    - table_div: Div
```

Comment:
@carloslimasd please comment this out so that we aren't shipping this doc example with this story (but we don't have to delete your other doc work that we will eventually use)